### PR TITLE
PMM-7083 : change aws staging start to node based start

### PR DESCRIPTION
**Pull request by AlyHKafoury**

### CODE REVIEW

File: pmm/aws-staging-start.groovy
Comment (EvgeniyPatlan) [inline pmm/aws-staging-start.groovy:347]: why do you need sudo for running docker?

File: pmm/aws-staging-start.groovy
Comment (EvgeniyPatlan) [inline pmm/aws-staging-start.groovy:424]:
```suggestion
                            sudo percona-release enable-only original testing
```

File: pmm/aws-staging-start.groovy
Comment (EvgeniyPatlan) [inline pmm/aws-staging-start.groovy:434]:
```suggestion
                            sudo percona-release enable-only original testing
```

File: pmm/aws-staging-start.groovy
Comment (EvgeniyPatlan) [inline pmm/aws-staging-start.groovy:439]:
```suggestion
                            sudo percona-release enable-only original testing
```

File: pmm/aws-staging-start.groovy
Comment (EvgeniyPatlan) [inline pmm/aws-staging-start.groovy:443]:
```suggestion
                            sudo percona-release enable-only original testing
```

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]
please provide link to the successfully finished job based on this pipeline

File: pmm/aws-staging-start.groovy
Comment (atymchuk) [inline pmm/aws-staging-start.groovy:209]: Isn't this your personal credential?

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]
where is link to successful build with this pipeline

File: pmm/aws-staging-start.groovy
Comment (AlyHKafoury) [reply pmm/aws-staging-start.groovy:209]: no this is commands to provision the instance you can echo them into `base64 -D` to decode them into the commands

**Comment by AlyHKafoury** [comment]
> where is link to successful build with this pipeline

@EvgeniyPatlan 
https://pmm.cd.percona.com/job/aws-staging-start/25296/console

**Review by EvgeniyPatlan — APPROVED** [review]
LGTM but please don't merge without @hors 's approval

File: pmm/aws-staging-start.groovy
Comment (hors) [reply pmm/aws-staging-start.groovy:209]: why can not you do it via ?
https://pmm.cd.percona.com/configureClouds/
label awscli  -> Init script

Comment (AlyHKafoury) [reply pmm/aws-staging-start.groovy:209]: this is for nodes we create on the fly for QA engineers to use it is not jenkins slaves

**Review by atymchuk — APPROVED** [review]

File: pmm/aws-staging-start.groovy
Comment (hors) [inline pmm/aws-staging-start.groovy:208]: As I told earlier it is very old amazon linux image based on centos 6,  maybe if you update it to  amazon linux 2 which is based on centos 7 a lot of your issues will be fixed. Anyway you need to do it sooner or later.

Comment (atymchuk) [reply pmm/aws-staging-start.groovy:208]: Good point Slava, but we need this pipeline working. So we will have to address this upgrade in a separate PR, which will be dedicated to migration and maybe some refactoring thereof.

Comment (atymchuk) [reply pmm/aws-staging-start.groovy:208]: @hors Which image would be the best replacement for `ami-15e9c770`?

Comment (hors) [reply pmm/aws-staging-start.groovy:208]: @atymchuk try to find Amazon Linux 2 suitable for your jenkins (I mean in your region) using https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#LaunchInstanceWizard e.g. this one ami-09558250a3419e7d0

File: pmm/aws-staging-start.groovy
Comment (hors) [inline pmm/aws-staging-start.groovy:277]: Why do we need to install java  in these two places ?

https://github.com/Percona-Lab/jenkins-pipelines/pull/684/files#diff-cf39489e2a5e3b641bec0e09fa1ee7a4f9f4c076eb1b8a7aa517440fb2e22a9cR209
https://github.com/Percona-Lab/jenkins-pipelines/pull/684/files#diff-cf39489e2a5e3b641bec0e09fa1ee7a4f9f4c076eb1b8a7aa517440fb2e22a9cR277

And why do we need to do it at all?  We are connecting to this remote instance  via ssh and it is not jenkins worker. 

Also I do not see a good commit message in this PR. What was the root of issue? What do we try to fix? 

As for me for now it seems as a workaround but not a general fix but I can be wrong. And as I know it is pretty urgent so please update the description and commit and then re-check everything without any rush.

**Review by hors — APPROVED** [review]

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**
